src/static/css/index.css

```diff
@@ -131,6 +132,15 @@ p {
   width: 40vw;
 }
 
+.intro-image-2020 {
```

Comment:
WDYT about renaming this element to be year-agnostic and using the `.year-2020` class to specify how 2020 styles should be applied? For example:

```css
.year-2020 .intro-image { ... }
```

This affects 2019.

Reply:
As per above this should be part of #894

Reply:
Fixed in https://github.com/HTTPArchive/almanac.httparchive.org/pull/893/commits/b5351ed6b55a2e02432f50f99765910e1c794ae9